qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -28,7 +27,7 @@ import java.util.Locale
 
 class MapboxNavigationViewCustomizedActivity : AppCompatActivity() {
 
-    private val routeLineOptions: MapboxRouteLineOptions by lazy {
+    private val myRouteLineOptions: MapboxRouteLineOptions by lazy {
```

Comment:
```suggestion
    private val routeLineOptions: MapboxRouteLineOptions by lazy {
```

please stay consistent with everything :)

Reply:
That's our internal, throwaway, test app. This correct. If it really bothers you go ahead and change it. All commits are welcome.

Reply:
Can we please keep this as it is?
Otherwise we will be forced to update it's usage to this monstrosity.
```
this@MapboxNavigationViewCustomizedActivity.routeLineOptions
```